Tests/SnapshotTestingTests/SnapshotTestingTests.swift

```diff
@@ -1024,6 +1024,80 @@ final class SnapshotTestingTests: XCTestCase {
     #endif
   }
 
+  #if os(iOS) || os(macOS)
+  final class ManipulatingWKWebViewNavigationDelegate: NSObject, WKNavigationDelegate {
+    func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
+      webView.evaluateJavaScript("document.body.children[0].classList.remove(\"hero\")") // Change layout
+    }
+  }
+  func testWebViewWithManipulatingNavigationDelegate() throws {
+    let manipulatingWKWebViewNavigationDelegate = ManipulatingWKWebViewNavigationDelegate()
+    let webView = WKWebView()
+    webView.navigationDelegate = manipulatingWKWebViewNavigationDelegate
+
+    let fixtureUrl = URL(fileURLWithPath: String(#file), isDirectory: false)
+      .deletingLastPathComponent()
+      .appendingPathComponent("__Fixtures__/pointfree.html")
+    let html = try String(contentsOf: fixtureUrl)
+    webView.loadHTMLString(html, baseURL: nil)
+    if !ProcessInfo.processInfo.environment.keys.contains("GITHUB_WORKFLOW") {
+      assertSnapshot(
+        matching: webView,
+        as: .image(size: .init(width: 800, height: 600)),
+        named: platform
+      )
+    }
+    _ = manipulatingWKWebViewNavigationDelegate
+  }
+
+  #if os(iOS) || os(macOS)
+  func testWebViewWithRealUrl() throws {
+    let manipulatingWKWebViewNavigationDelegate = ManipulatingWKWebViewNavigationDelegate()
+    let webView = WKWebView()
+    webView.navigationDelegate = manipulatingWKWebViewNavigationDelegate
+
+    webView.load(URLRequest(url: URL(string: "https://www.pointfree.co")!))
```

Comment:
This loads an actual URL. Is that something you want to prevent in the unit tests? I can image you want to avoid it, but it's the best real life example..